go/engine/engine.go

```diff
@@ -61,6 +77,21 @@ func runPrereqs(e Engine, ctx *Context) (err error) {
 
 }
 
+func tryStandaloneLogin(e Engine, ctx *Context) error {
+	if !e.G().Env.GetStandalone() {
```

Comment:
No, sorry it is confusing.  tryStandaloneLogin is only called after the prereq check fails.  And if it returns an error, then the prereq check returns the original error, not this error.